tests/MODULE.bazel

```diff
@@ -1,6 +1,156 @@
-###############################################################################
-# Bazel now uses Bzlmod by default to manage external dependencies.
-# Please consider migrating your external dependencies from WORKSPACE to MODULE.bazel.
-#
-# For more details, please check https://github.com/bazelbuild/bazel/issues/18958
-###############################################################################
+module(name = "rules_scala_annex_tests")
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "rules_java", version = "7.12.2")
+bazel_dep(name = "rules_jvm_external", version = "6.5")
+bazel_dep(name = "rules_scala_annex")
+local_path_override(
+    module_name = "rules_scala_annex",
+    path = "..",
+)
+
+register_toolchains(
+    "//:repository_default_toolchain_21_definition",
+    "//dependencies/indirect:test_zinc_2_13_direct_deps_off",
+    "//dependencies/indirect:test_zinc_2_13_used_deps_off",
+    "//plugins/kind-projector:test_zinc_2_13_with_kind_projector",
+    "//plugins/semanticdb:scala_2_13_with_semanticdb",
+    "//plugins/semanticdb:scala_3_with_semanticdb",
+    "//scala:test_bootstrap_2_12",
+    "//scala:test_bootstrap_2_13",
+    "//scala:test_bootstrap_3",
+    "//scala:test_zinc_2_12",
+    "//scala:test_zinc_2_13",
+    "//scala:test_zinc_3",
+    "//scala:test_zinc_2_13_fatal_deprecation_opt",
+    "//scala:test_zinc_2_13_log_level_debug",
+)
+
+# Please ensure these stay up-to-date with the versions in `rules/scala/versions.bzl`.
```

Comment:
Should the path here be updated to refer to the parent workspace instead of the test workspace?

Reply:
Can we also update this comment to ask people to update the versions in annex's MODULE.bazel?

Reply:
> Should the path here be updated to refer to the parent workspace instead of the test workspace?

Doesn't `rules/scala/versions.bzl` imply it's in the main workspace because `tests/rules` doesn't exist? I could make it a relative path if you'd like

> Can we also update this comment to ask people to update the versions in annex's MODULE.bazel?

I figured that rather than having every set of variables point to all the others (`O(N^2)`), we could have every set point to `versions.bzl` and have that point to all the others (`O(N)`).

Reply:
That's fair. I'll update them to reference each other.